tests/test_animator.py

```diff
@@ -7,14 +7,11 @@
 import grpc
 import pytest
 
-#def test_init_args():
-#    AnimationArgs()
-
 animation_prompts={0:"foo bar"}
 
```

Comment:
add tests for new Animator methods/functions

Reply:
well they're internal methods. I added MockStub to test_client.py allowing testing of the Api methods with exception of transform. when it's fully fleshed out can use to do a full test of Animator against it. don't want to spend a ton of time implemention mock and tests for transform APIs though since they await ChainRequest implementation and may change quite a bit.